100_essentials/greetings/test_greetings.py

```diff
@@ -0,0 +1,55 @@
+from inspect import getfullargspec, signature
+import unittest
+
+from greetings import greetings
+
+
+class GreetingsTest(unittest.TestCase):
+    def test_required_args(self):
+        argsspec = getfullargspec(greetings)
```

Comment:
Hell yeah `getfullargspec`. I actually don't know that I would have even thought of that.